### PR TITLE
docs(operators): add documentation for window

### LINKED ISSUE #124: Add docs for operator : transformation - window
Add docs for operator window under group transformation 
 
Below is minimum details needed 
 
- [ ] name 
- [ ] operatorType 
- [ ] signature 
- [ ] parameters 
- [ ] short description 
- [ ] Technical description 
- [ ] Visual example  (Marble diagram) 
- [ ] Real world example (Non-framework) 
- [ ] Related Operators

### PR DESCRIPTION
**Pull request by hardikpthv**
Closes #124

### CODE REVIEW

**Comment by ladyleet** [comment]
thanks @hardikpthv!

File: src/operator-docs/transformation/window.ts [hunk 1]
```diff
@@ -1,6 +1,68 @@
 import { OperatorDoc } from '../operator.model';
 
 export const windowOperator: OperatorDoc = {
-  'name': 'window',
-  'operatorType': 'transformation'
+  name: 'window',
+  operatorType: 'transformation',
+  signature:
+    'public window(windowBoundaries: Observable<any>): Observable<Observable<T>>',
```
Comment (btroncone) [inline src/operator-docs/transformation/window.ts]: We can remove the generic signatures here, can be done later in one go though. Don't think it's worth holding up the PR.

**Comment by hardikpthv** [comment]
Pleasure @ladyleet :)

**Comment by petebacondarwin** [comment]
For some reason this PR added a submodule which points to the same repo. Not sure that was desired.

**Comment by hardikpthv** [comment]
@petebacondarwin yeah I see. Let me fix this in next PR.

**Pull request merged**